main-app/app/html/inGame.html

```diff
@@ -13,46 +13,60 @@
 <div class="gameAbout" ng-controller="MainController">
 
     <img src="images/bingo90.png">
+<br>
+    <div class="callRow">
+        <table width="460px">
+            <tr>
+                <td width="250px">
+                    <div class="lastCall">{{bingoCall.lastCallsDisplay[4]}}</div>
+                    <div class="lastCall">{{bingoCall.lastCallsDisplay[3]}}</div>
+                    <div class="lastCall">{{bingoCall.lastCallsDisplay[2]}}</div>
+                    <div class="lastCall">{{bingoCall.lastCallsDisplay[1]}}</div>
+                    <div class="lastCall">{{bingoCall.lastCallsDisplay[0]}}</div>
+                </td>
+                <td width="62px">
+                    <div class="currentCall">{{bingoCall.callMessage.payload.call}}</div>
+                </td>
+                <td width="148px">
+                    Game ID: {{gameApi.returnedMessage.payload.gameId}} Call: {{bingoCall.callMessage.payload.callnumber}}
+                </td>
+            </tr>
+        </table>
+    </div>
 
-    <h3>Game ID: {{gameModel.returnedMessage.payload.gameId}}</h3>
-
-    <br>
-    Current Number: {{bingoCall.callMessage.payload.call}} <br>
-    Call Number: {{bingoCall.callMessage.payload.callnumber}}
-
-        <div class="bingoBackground">
+    <div class="bingoBackground">
             <div class="gameboard">
-                <div ticket-square="0" class="bingoCell textCenter"></div>
-                <div ticket-square="1" class="bingoCell textCenter"></div>
-                <div ticket-square="2" class="bingoCell textCenter"></div>
-                <div ticket-square="3" class="bingoCell textCenter"></div>
-                <div ticket-square="4" class="bingoCell textCenter"></div>
-                <div ticket-square="5" class="bingoCell textCenter"></div>
-                <div ticket-square="6" class="bingoCell textCenter"></div>
-                <div ticket-square="7" class="bingoCell textCenter"></div>
-                <div ticket-square="8" class="bingoCell textCenter"></div>
+                <div ticket-square="0" class="textCenter"></div>
```

Comment:
These squares look like a good candidate for a directive in later braches (No action in this branch)